code/game/objects/items/control_wand.dm

```diff
@@ -155,7 +155,78 @@
 	busy = FALSE
 	icon_state = "hacktool"
 
+/// How long before you can "jangle" your keyring again (to prevent spam)
+#define JANGLE_COOLDOWN 10 SECONDS
+
+/obj/item/door_remote/janikeyring
+	name = "janitor's keyring"
+	desc = "An absolutely unwieldy set of keys attached to a metal ring."
```

Comment:
Since janitor is a more newbie-friendly job, think it might make sense to provide a bit of instruction on how to use this new equipment?

Reply:
done